airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py

```diff
@@ -2,101 +2,133 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
 
-import responses
+import json
+import os
+from typing import Any, Mapping
+
 from airbyte_cdk.models import SyncMode
-from source_pivotal_tracker.source import Activity, Epics, Labels, PivotalAuthenticator, ProjectMemberships, Projects, Releases, Stories
+from airbyte_cdk.sources.streams import Stream
+from pytest import fixture
```

Comment:
Please remove this import as it's not used here

Reply:
Done